db/seeds.rb

```diff
@@ -14,15 +13,13 @@
 Topic.find_or_create_by(name: "health", title: "Health")
 Topic.find_or_create_by(name: "transport", title: "Transport")
 
-puts "Seeding organisations"
 org = Organisation.find_or_create_by(
   uuid: "90aefa0d-0e92-4895-a7fd-c1adb2b3f14f",
   name: "government-digital-service")
 
-org.update_attribute(:govuk_content_id, "af07d5a5-df63-4ddc-9383-6a666845ebe9")
-org.update_attribute(:title, "Government Digital Service")
+org.update(govuk_content_id: "af07d5a5-df63-4ddc-9383-6a666845ebe9")
+org.update(title: "Government Digital Service")
```

Comment:
It might be nice to combine these into one.

Reply:
Sure, done 👍